src/components/Product.vue

```diff
@@ -3,6 +3,8 @@
     <h1>{{ currentProduct }}</h1>
     <h2>Price: ${{ price.toFixed(2) }}</h2>
     <img :src="img">
+    <br>
```

Comment:
Its better to use use `<br>` in paragraph text when you need to break lines of text. In this case, it would be better to wrap the Summary in a `<div>` or `<span>` and use CSS to apply the styling needed.